lib/queue.rb

```diff
@@ -1,28 +1,33 @@
 class Queue
```

Comment:
This works, but you were asked to implement this using a circular buffer.  I think you missed a bit about the point of the assignment and the last part of last week's class.